api/tests/exceptions/test_exception_handlers.py

```diff
@@ -25,5 +25,5 @@ def get_app_config_raising_generic_exception():
     monkeypatch.setitem(app.view_functions, 'get_app_config_', get_app_config_raising_generic_exception)
     response = client.get('/manager/get_app_config')
 
-    assert response.status_code == 500
-    assert response.json == {'code': 500, 'message': 'Something went wrong'}
+    assert response.status_code == 400
+    assert response.json == {'code': 400, 'message': 'Something went wrong'}
```

Comment:
nitpick: I think we decided with a different error message at the time which was a bit more friendly?